lib/plist-transform.js

```diff
@@ -2,9 +2,8 @@
 const path = require('path');
 const fs = require('fs');
 const plist = require('plist');
-const pify = require('pify');
-
-const fsP = pify(fs);
+const read = require('read-glob-promise');
```

Comment:
Actually, I realised that since npm already copies the contents of the README to `readme` on install, that I could just grab it from there… which makes this _much_ simpler and no need for an extra dependency anyway. I'll definitely check out _globby_ in future though since it's a common need. 👍